### PR TITLE
scroll to top icon

### PR DESCRIPTION
**Pull request by adsingh14**
Added Scroll-to-Top icon without text.

Issue:

### CODE REVIEW

File: src/components/icon/_scroll-to-top.less [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    text-rendering: auto;
+    -webkit-font-smoothing: antialiased;
+  }
+
+  &:not(.rounded-btn) {
+    padding: 0 0.5em;
+  }
+
+  &.rounded-btn {
+    width: 2em;
```
Comment (shahednasser) [inline src/components/icon/_scroll-to-top.less:18]: change width and height to 1.5em and padding to `0 0 0 1px`

**Review by shahednasser — CHANGES_REQUESTED** [review]
I just have one change to what you've done.

File: src/components/icon/_scroll-to-top.less (hunk 1 above)
Comment (adsingh14) [reply src/components/icon/_scroll-to-top.less]: Fixed !

**Comment by shahednasser** [comment]
Great thank you!

@all-contributors please add @adsingh14 for code

**Pull request merged**